admin/sql/updates/20190317-mbs-9941-mbs-10062.sql

```diff
@@ -0,0 +1,454 @@
+\set ON_ERROR_STOP 1
+BEGIN;
+
+CREATE TABLE genre ( -- replicate (verbose)
+    id                  SERIAL, -- PK
+    name                VARCHAR NOT NULL,
+    comment             VARCHAR(255) NOT NULL DEFAULT '',
+    gid                 uuid NOT NULL
+);
+
+CREATE TABLE genre_alias ( -- replicate (verbose)
+    id                  SERIAL,
+    genre               INTEGER NOT NULL, -- references genre.id
+    name                VARCHAR NOT NULL,
+    locale              TEXT,
+    edits_pending       INTEGER NOT NULL DEFAULT 0 CHECK (edits_pending >= 0),
+    last_updated        TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
+    primary_for_locale  BOOLEAN NOT NULL DEFAULT false,
+    CONSTRAINT primary_check CHECK ((locale IS NULL AND primary_for_locale IS FALSE) OR (locale IS NOT NULL)),
+);
+
+ALTER TABLE genre ADD CONSTRAINT genre_pkey PRIMARY KEY (id);
+
+ALTER TABLE genre_alias ADD CONSTRAINT genre_alias_pkey PRIMARY KEY (id);
+
+CREATE UNIQUE INDEX genre_idx_gid ON genre (gid);
+CREATE UNIQUE INDEX genre_idx_name ON genre (LOWER(name));
+
+CREATE INDEX genre_alias_idx_genre ON genre_alias (genre);
+CREATE UNIQUE INDEX genre_alias_idx_primary ON genre_alias (genre, locale) WHERE primary_for_locale = TRUE AND locale IS NOT NULL;
+
+INSERT INTO genre (gid, name)
+VALUES  ('54c01942-22fd-4184-9877-1db0089b18f1', 'acid house'),
```

Comment:
I'll admit pre-generating the gids seems weird, but that's probably because we've never added a new entity that was seeded like this before. I think v4 UUIDs do make sense though, since genres come from the outside world and have no unique name guarantee.